lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/EvaluatorRequestorImpl.java

```diff
@@ -72,7 +72,7 @@ public synchronized void submit(final EvaluatorRequest req) {
     }
 
     try (LoggingScope ls = loggingScopeFactory.evaluatorSubmit(req.getNumber())) {
-      final DriverRuntimeProtocol.ResourceRequestProto.Builder request = DriverRuntimeProtocol.ResourceRequestProto
+      final ResourceRequestEventImpl.Builder request = ResourceRequestEventImpl
```

Comment:
We could use the `EvaluatorRequest` as the type instead of adding another event type.